WSTEAMA-1243-p2 Media promos on home pages do not read well without CSS

Nabeel1276 opened the pull request:
Resolves JIRA [WSTEAMA-1243](https://jira.dev.bbc.co.uk/browse/WSTEAMA-1243)

Before:
![image](https://github.com/user-attachments/assets/61c33aec-83cf-4c0a-9361-7516f9d0cc57)

After
![image](https://github.com/user-attachments/assets/42392c7f-951d-4480-a8f7-045b187d9016)


Code changes
======

- Used template strings/literals to add a comma and a space between the last word of the title and the word 'Duration' on a media promo


Testing
======

1. Visit the afrique home page
2. Disable all CSS using Web developer chrome extension
3. Locate media promos 
4. There should be a comma followed by a space before the word "duration"

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

Nabeel1276 committed:
added space between the last word of the title and the word duration

Nabeel1276 committed:
added comma and space before duration string and updated unit tests to reflect it

Nabeel1276 committed:
added comma and space before duration string and updated unit tests to reflect it

greenc05 commented on src/app/components/Curation/HierarchicalGrid/index.test.tsx:
```diff
@@ -63,7 +63,7 @@ describe('Hierarchical Grid Curation', () => {
     );
 
-    const durationString = 'Duration, 2,03';
+    const durationString = ', Duration, 2,03';
```
We shouldn't have a comma after the word 'Duration'

greenc05 reviewed (commented):
Thanks @Nabeel1276 are you please able to add a screen shot of the updated component without CSS to the PR?

Nabeel1276 commented:
> Thanks @Nabeel1276 are you please able to add a screen shot of the updated component without CSS to the PR?

Sure, do you mean the way the screenshot is in the ticket?

greenc05 commented:
> > Thanks @Nabeel1276 are you please able to add a screen shot of the updated component without CSS to the PR?
> 
> Sure, do you mean the way the screenshot is in the ticket?

Yes please, though we may need to look at removing the comma after the word 'Duration' first if that is still here, on the ticket this is referenced here: 'I think the comma has been put in the wrong place after the word Duration, see screen shot.' The ticket isn't very well written, sorry about that.

Nabeel1276 committed:
removed unnecessary comma after duration

Nabeel1276 commented:
> > > Thanks @Nabeel1276 are you please able to add a screen shot of the updated component without CSS to the PR?
> > 
> > 
> > Sure, do you mean the way the screenshot is in the ticket?
> 
> Yes please, though we may need to look at removing the comma after the word 'Duration' first if that is still here, on the ticket this is referenced here: 'I think the comma has been put in the wrong place after the word Duration, see screen shot.' The ticket isn't very well written, sorry about that.

Not a problem, I have removed the redundant comma and added the screenshot of the media promo with the correct changes.

karinathomasbbc commented on src/app/components/Curation/CurationPromo/index.test.tsx:
```diff
@@ -44,7 +44,7 @@ describe('Curation Promo', () => {
     );
 
-    const durationString = 'Duration, 2,03';
+    const durationString = ', Duration 2,03';
```
Can we remove the leading `,` because the tests should work without it? (And is more readable)
```suggestion
    const durationString = 'Duration 2,03';
```

karinathomasbbc replied:
Turns out the tests fail if we remove the comma, so best to leave as is!

karinathomasbbc commented on src/app/components/Curation/HierarchicalGrid/index.test.tsx:
```diff
@@ -63,7 +63,7 @@ describe('Hierarchical Grid Curation', () => {
     );
 
-    const durationString = 'Duration, 2,03';
+    const durationString = ', Duration 2,03';
```
As above, remove leading `,`

karinathomasbbc replied:
As above, tests fail, due to the fact that the test is fetching the element by its text (and it must match exactly). This change is required.

Nabeel1276 committed:
Merge branch 'latest' into WSTEAMA-1243-p2-media-promos-on-home-pages-do-not-read-well-without-css

LilyL0u committed:
Merge branch 'latest' into WSTEAMA-1243-p2-media-promos-on-home-pages-do-not-read-well-without-css

Nabeel1276 committed:
Merge branch 'latest' into WSTEAMA-1243-p2-media-promos-on-home-pages-do-not-read-well-without-css

Nabeel1276 committed:
Merge branch 'latest' into WSTEAMA-1243-p2-media-promos-on-home-pages-do-not-read-well-without-css

Nabeel1276 committed:
Merge branch 'latest' into WSTEAMA-1243-p2-media-promos-on-home-pages-do-not-read-well-without-css

Nabeel1276 committed:
Merge branch 'latest' into WSTEAMA-1243-p2-media-promos-on-home-pages-do-not-read-well-without-css

Nabeel1276 committed:
Merge branch 'latest' into WSTEAMA-1243-p2-media-promos-on-home-pages-do-not-read-well-without-css

LilyL0u committed:
Merge branch 'latest' into WSTEAMA-1243-p2-media-promos-on-home-pages-do-not-read-well-without-css
